.gitignore

```diff
@@ -16,6 +16,7 @@
 /npm-debug.log*
 /testem.log
 /yarn-error.log
+/.vscode
```

Comment:
Personally I don't think this should be ignored. `/.vscode` allows sharing project specific settings for VSCode. You can of course override them :)

Reply:
Removed in e7c85a7.